modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -4557,6 +4557,9 @@ GDScriptParser::DataType GDScriptAnalyzer::type_from_property(const PropertyInfo
 						result.is_constant = false;
 					}
 				}
+			} else if ((p_property.usage & PROPERTY_USAGE_CLASS_IS_BITFIELD) && p_property.class_name != StringName()) {
+				// BitField isn't supported (yet?).
+				result.is_constant = false;
```

Comment:
This `else` is not needed. The only reason it sets `result.is_constant = false` here is because `make_enum_type()` makes it `true`, but in the `else` branch it is not called.